content/doc/tutorials/tutorial-for-installing-jenkins-on-IBM-Cloud.adoc

```diff
@@ -7,200 +7,250 @@ section: doc
+
+. On the application page, select the _empty dot_ next to the cluster you wish to use.
++
+image::tutorials/IBMCloud/cluster-select.png[IBM cluster selection,role="screenshot"]
+
+. Select *Target namespace* and then choose the default Namespace or use a custom one.
++
+image::tutorials/IBMCloud/details-namespace.png[IBM space name,role="screenshot"]
+
+. Give a unique *name* to the workspace, that you can easily recognize.
++
+image::tutorials/IBMCloud/details-names.png[IBM workspace name,role="screenshot"]
+
+. Select the *Resource group* you want to use.
+This is for access control and billing purposes.
+For more information, please refer to link:https://cloud.ibm.com/docs/account?topic=account-account_setup#bp_resourcegroups[resource groups].
++
+image::tutorials/IBMCloud/details-resource.png[Resource details,role="screenshot"]
+
+. Assign *tags* to your Jenkins, for more information, refer to link:https://cloud.ibm.com/docs/account?topic=account-tag[tags].
++
+image::tutorials/IBMCloud/details-tags.png[Default value parameters,role="screenshot"]
+
+. Select *Parameters with default values*.
+You can set custom deployment values or use the default ones.
++
+image::tutorials/IBMCloud/parameters.png[IBM parameters setup,role="screenshot"]
+
+. Set the Jenkins password in the parameters.
++
+image::tutorials/IBMCloud/password.png[Create password instructions,role="screenshot"]
+
+. Once complete, acknowledge the license agreement and select *Install*.
++
+image::tutorials/IBMCloud/install.png[Installation instructions,role="screenshot"]
++
+The Jenkins workspace begins installing. Wait for the installation to complete.
++
+image::tutorials/IBMCloud/in-progress.png[Workspace installation screen,role="screenshot"]
++
+Once complete, your Jenkins workspace is now successfully deployed.
++
+image::tutorials/IBMCloud/done.png[IBM workspace deployment screen,role="screenshot"]
+
+== Verifying the Jenkins installation
+
+After deploying Jenkins on your cluster, you can verify the Jenkins installation.
+
+To verify the Jenkins installation:
+
+. Go to link:http://cloud.ibm.com/resources[Resources] in your browser.
+. Select *Clusters*.
+. Select your cluster.
++
+image::tutorials/IBMCloud/resource-select.png[Resource selection screen,role="screenshot"]
++
+Now you are at your cluster overview.
+. Select *Actions* in the top right corner of the page.
+. Select *Web terminal* in the dropdown menu.
++
+.Cluster overview page
+image::tutorials/IBMCloud/cluster-main.png[Clusters overview page,role="screenshot"]
+
+. Select *Install* and then wait for the installation to complete.
++
+image::tutorials/IBMCloud/terminal-install.jpg[terminal install screen,role="screenshot"]
+
+. Once installation has completed, you must repeat this process.
+
+.. Select *Actions*.
+.. Select *Web terminal* and a terminal window opens.
+
+. *Type* in the terminal.
+You must change `NAMESPACE` to the namespace you choose during the deployment setup.
```

Comment:
I believe this should be past tense, since they already chose the namespace during the deployment setup.
```suggestion
You must change `NAMESPACE` to the namespace you chose during the deployment setup.
```